<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_7" columnstretch="80,20">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="5,95,0">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Plano z =</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSliderPlanoZ">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Areadedesenho" name="widget" native="true">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_6" rowstretch="80,30">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Esfera</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="horizontalSliderRaioEsfera">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Dimens√µes da caixa</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QSlider" name="SliderXbox">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QSlider" name="SliderYbox">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>z</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QSlider" name="SliderZbox">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Cor:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="horizontalSliderR">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>G</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="horizontalSliderG">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0" rowspan="2" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSlider" name="horizontalSliderB">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar_2"/>
    <addaction name="actionFechar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovo_Arquivo"/>
   <addaction name="actionPutvoxel"/>
   <addaction name="actionCutvoxel"/>
   <addaction name="actionPutsphere"/>
   <addaction name="actionCutsphere"/>
   <addaction name="actionPutbox"/>
   <addaction name="actionCutbox"/>
  </widget>
  <action name="actionFechar">
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionSalvar_2">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionPutvoxel">
   <property name="text">
    <string>Putvoxel</string>
   </property>
  </action>
  <action name="actionCutvoxel">
   <property name="text">
    <string>Cutvoxel</string>
   </property>
  </action>
  <action name="actionPutbox">
   <property name="text">
    <string>Putbox</string>
   </property>
  </action>
  <action name="actionCutbox">
   <property name="text">
    <string>Cutbox</string>
   </property>
  </action>
  <action name="actionPutsphere">
   <property name="text">
    <string>Putsphere</string>
   </property>
  </action>
  <action name="actionCutsphere">
   <property name="text">
    <string>Cutsphere</string>
   </property>
  </action>
  <action name="actionNovo_Arquivo">
   <property name="text">
    <string>Novo Arquivo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Areadedesenho</class>
   <extends>QWidget</extends>
   <header location="global">areadedesenho.h</header>
   <container>1</container>
   <slots>
    <signal>triggered(bool)</signal>
    <slot>setdimx(int)</slot>
    <slot>setdimy(int)</slot>
    <slot>setR(int)</slot>
    <slot>setG(int)</slot>
    <slot>setB(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderPlanoZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setR(int)</slot>
  <slot>setG(int)</slot>
  <slot>setB(int)</slot>
  <slot>putSphere()</slot>
 </slots>
</ui>
